package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -152,6 +152,7 @@
 """
 import logging
 import warnings
+from MDAnalysis import AtomGroup
```

Comment:
Imports are usually grouped as "python built-ins", "external libraries", "internal library", see the PEP8 guide for more details: https://peps.python.org/pep-0008/#source-file-encoding

Can you move this to after line 159 please?